jerry-port/default/default-date.c

```diff
@@ -74,3 +83,21 @@ double jerry_port_get_current_time (void)
   return 0.0;
 #endif /* __GNUC__ */
 } /* jerry_port_get_current_time */
+
+#ifdef JERRY_DEBUGGER
+#include <jrt.h>
+
```

Comment:
You've added this include twice. Remove from here.

Reply:
why is jrt.h required in default-date.c at all? that seems like a layering violation to me. if that's only for the sake of uint32_t, there are standard libc headers that could be included.